src/models/Deck.ts

```diff
@@ -45,7 +49,7 @@ export class Deck {
   // rather than use Array.sort(), which is predictable even when using Math.random(),
   // we will create our own random sorting algorithm
   shuffle() {
-    for (let i = this.numberOfCards - 1; i > 0; i--) {
+    for (let i = this.deckSize - 1; i > 0; i--) {
```

Comment:
We should pick either `this.deckSize` or `this.cards.length` for consistency throughout the file.

Reply:
Updated to `this.deckSize`.